core/Plugin/Report.php

```diff
@@ -680,6 +680,16 @@ public function getCategory()
     }
 
     /**
+     * Get the translation key of the category the report belongs to.
+     * @return string
+     * @ignore
+     */
+    public function getCategoryKey()
+    {
+        return $this->category;
+    }
```

Comment:
See 3 lines above, `getCategory` currently translates the category. We will hopefully change that with Piwik 3.0.